packages/amplify-provider-awscloudformation/lib/push-resources.js

```diff
@@ -18,6 +18,7 @@ const archiver = require('../src/utils/archiver');
 const amplifyServiceManager = require('./amplify-service-manager');
 const ziparchiver = require('archiver');
 const glob = require('glob');
+const { packageLayer } = require('../../amplify-category-function/lib/provider-utils/awscloudformation/utils/functionPluginLoader');
```

Comment:
see other comment about the packageLayer function. TLDR, we shouldn't be importing based on a path. just import from the `amplify-category-function` module